frontend/src/app/disputes/[id]/page.tsx

```diff
@@ -53,13 +55,15 @@ export default async function DisputePage({ params }: Props) {
   if (error || !data) {
     return <h1>{error}</h1>;
   }
-
+  var eviStr : string = data.evidence[0].toString()
+  eviStr = eviStr.split("/").pop() as string
```

Comment:
What about the rest of the evidence array?

Reply:
Well when we can upload more than 1 evidence dock I'll change the functionality, shouldn't be too much